enhancements/ingress/optional-ingress.md

```diff
@@ -0,0 +1,274 @@
+---
+title: optional-ingress
+authors:
+  - "@alebedev87"
+reviewers:
+  - "@Miciah"
+  - "@bparees"
+  - "@cewong"
+  - "@wking"
+approvers:
+  - "@Miciah"
+  - "@bparees"
+api-approvers:
+  - N/A
+creation-date: 2023-05-30
+last-updated: 2023-05-30
+tracking-link:
+  - https://issues.redhat.com/browse/NE-1129
+see-also:
+  - "/enhancements/installer/component-selection.md"
+  - "/enhancements/ingress/aws-load-balancer-operator.md"
+replaces:
+superseded-by:
+---
+
+# Make Ingress Operator optional
+
+## Summary
+
+This enhancement describes the implementation of a new capability to enable/disable the default ingress of the OpenShift cluster.
+The new capability (from here on called the ingress capability) includes [the cluster ingress operator](https://github.com/openshift/cluster-ingress-operator) as well as the custom resources managed by the operator.
+This enhancement focuses on the requirements for [RFE-3395](https://issues.redhat.com/browse/RFE-3395) which targets ROSA, a managed OpenShift service based on HyperShift. The implementation of the ingress capability on the standalone OpenShift is out of the scope of this enhancement.
+
+## Motivation
+
+Users of the ROSA managed OpenShift service would like to use [AWS load balancers](https://github.com/openshift/enhancements/blob/master/enhancements/ingress/aws-load-balancer-operator.md) as a way to accept the ingress traffic into the cluster.
+The cluster ingress operator becomes unnecessary and can be disabled to reduce the resource consumption.
```

Comment:
nit: it's a bit beyond resource consumption.  HyperShift currently runs the ingress router on NodePool compute nodes, and not on the management cluster.  Those compute nodes are further from the control of [the Cluster Service Provider][1].  Moving to an implementation that does not require visible-outside-of-AWS hosting removes one of the loads that is infra-hosted on standalone clusters, and brings zero-compute closer.

[1]: https://hypershift-docs.netlify.app/reference/concepts-and-personas/#personas

Reply:
I'm not sure I fully understood this comment.

The way I read it: an ability to disable a component which is a part of the hosted control plane but which has its operand running on the user's compute is a good thing. This enables the cluster instance users to make their ingress decisions on their compute making therefore the hosted control plane thinner.

Reply:
I updated this part to better detail the motivation. Let me know if I understood you correctly.

Reply:
Your 7247e12e text for this section looks good to me; feel free to mark this thread resolved :bow: